src/components/BodyWeight/index.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {render} from '@testing-library/react';
+import {get_weights} from "../../services";
```

Comment:
We could use absolute import here

Reply:
Yes. I'll see if where I can configure the ide because I always just click on "add Import"

Reply:
I already configured it in the tsconfig file. removing the leading ../../ should be enough.